src/Microsoft.Health.Fhir.Core/Features/TaskManagement/TaskInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.Core.Features.TaskManagement
+{
+    public class TaskInfo
+    {
+        public string TaskId { get; set; }
+
+        public string QueueId { get; set; }
```

Comment:
Just for my understanding, do we expect there will be multiple queues for each fhir-server instance?

Reply:
Yes, we want to support multiple queue. One typical scénario is like we have a orchestrator task and multiple sub tasks. Orchestrator task need high availability and less resource it is to create and monitor sub task. And sub task is highly resource consumer. We'd better have 2 different environment with different capacity. We can use different queueid to dispatch task to different environment. Currently we have only one queueid in FHIR server, and we plan to enable it in separate runtime environment.